default.py

```diff
@@ -240,6 +240,15 @@ def SEARCH_ALT(payload, params):
 
     return control.draw_items(search_res)
 
+@route('list_sources/*')
+def LIST_SOURCES(payload, params):
```

Comment:
I have the feeling that we can just use play and add some param to indicate autoplay should be off